packages/bpk-component-modal/src/BpkModal.tsx

```diff
@@ -37,7 +40,7 @@ export type Props = Partial<ModalDialogProps> & {
   isOpen: boolean;
   closeOnScrimClick?: boolean;
   closeOnEscPressed?: boolean;
-  renderTarget?: null | HTMLElement | (() => null | HTMLElement);
```

Comment:
Interesting that `() => null` default value satisfies the `() => HTMLElement` type and this is not needed 🤔

Reply:
This I had followed from our other components that seems to work. I guess its satisfied under the `null` part of the type